WalletWasabi.Fluent/Helpers/TextHelpers.cs

```diff
@@ -78,5 +78,15 @@ public static string ToFormattedString(this Money money)
 
 			return moneyString;
 		}
+
+		public static string RemoveExtraWhitespaces(this string text)
+		{
+			while (text.Contains("  "))
```

Comment:
Maybe we should also check for tabs ?

Spaces are now handled correctly.

Reply:
Maybe we can use: https://docs.microsoft.com/en-us/dotnet/api/system.char.iswhitespace?view=net-5.0

Reply:
I might choose the wrong function name. The goal is here to allow only 1 space char. I wonder I should rename the function to `ToCorrectedLabelFormat()` and put all logic there.

1. Clear whitespaces from the end.
2. Clear whitespaces from the start.
3. Only allow 1 whitespace in the mid.
4. Remove tabs and remove newlines.

Reply:
Maybe just call it ParseLabels

Here is some code I have used in other library for such function

        private static readonly Regex MultipleSpaces = new Regex(@" {2,}", RegexOptions.Compiled);


        /// <summary>
        /// Prepare the text according to the whitespace handling rules and text transformations.  <see href="http://www.w3.org/TR/SVG/text.html">SVG Spec</see>.
        /// </summary>
        /// <param name="value">Text to be prepared</param>
        /// <returns>Prepared text</returns>
        protected string PrepareText(string value)
        {
            value = ApplyTransformation(value);
            value = new StringBuilder(value).Replace("\r\n", " ").Replace('\r', ' ').Replace('\n', ' ').Replace('\t', ' ').ToString();
            return this.SpaceHandling == XmlSpaceHandling.Preserve ? value : MultipleSpaces.Replace(value.Trim(), " ");
        }

Reply:
I have found the simplest solution.
```
Regex.Replace(text, @"\s+", " ").Trim();
```